openy_af4_vue_app/src/components/WizardBar.vue

```diff
@@ -7,7 +7,12 @@
             <span class="filters">
               <a :class="{ 'no-filters': !filtersCount }" role="button" @click="showFiltersModal()">
                 <font-awesome-icon icon="filter" />
-                {{ filtersCount | formatPlural('1 Filter Applied', '@count Filters Applied') }}
+                <span v-if="filtersCount">
+                  {{ filtersCount | formatPlural('1 Filter Applied', '@count Filters Applied') }}
+                </span>
+                <span v-if="!filtersCount">
+                  {{ '0 Filter Applied' | t }}
```

Comment:
```suggestion
                  {{ 'No Filters Applied' | t }}
```

Reply:
@podarok yeah, but on the design uses `0 Filter`
![image](https://github.com/YCloudYUSA/yusaopeny_activity_finder/assets/744406/49d34906-0bda-492a-b5aa-98b3fd330a11)

Reply:
Ok, we live in a world of robots, not humans now